docs/src/modules/components/overview/WorldMapSvg.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { styled, useTheme } from '@mui/material/styles';
+import clsx from 'clsx';
+
+const Continent = styled('g')(({ theme }) => ({
+  cursor: 'pointer',
+  opacity: 0.9,
+  '&:hover ': { opacity: 1 },
+  '&.selected': {
+    fill: theme.palette.primary.main,
+  },
+}));
+
+export type ContinentClickHandler = (event: React.MouseEvent<SVGElement>, timezone: string) => void;
+
+type WorldMapSvgProps = {
+  onClickContinent: ContinentClickHandler;
+  selectedTimezone: string | null;
+};
+
+export default function WorldMapSvg({ onClickContinent, selectedTimezone }: WorldMapSvgProps) {
+  const brandingTheme = useTheme();
+
+  const getMapBaseColor = (lightness: number, opacity: number = 1) => {
+    return brandingTheme.palette.mode === 'light'
+      ? `hsla(210,20%,${lightness}%, ${opacity * 100}%)`
+      : `hsla(210,20%,${lightness}%, ${opacity * 100}%)`;
+  };
+
+  const timezones: { [key: string]: string } = {
+    europe: 'Europe/Paris',
+    asia: 'Asia/Hong_kong',
+    southAmerica: 'America/Sao_Paulo',
+    northAmerica: 'America/Chicago',
+    africa: 'Africa/Casablanca',
+    australia: 'Australia/Brisbane',
+  };
+  // TODO: simplify SVG
+  return (
+    <svg
```

Comment:
I will be replacing this with a more simplified map. The functionality will stay the same 👌